docs/docker.md

```diff
@@ -38,6 +38,10 @@ Create TeSS configuration files:
 
 `tess.yml` is used to configure features and branding of your TeSS instance. `secrets.yml` is used to hold API keys etc.
 
+Create docker image:
+
+    docker build -t taxila-app .
```

Comment:
let's keep this generic enough so the README can be the same as on TeSS (and redo this PR towards their repository, we can merge the changes back later)